spaceship.zsh

```diff
@@ -446,11 +448,20 @@ spaceship_host() {
 spaceship_dir() {
   [[ $SPACESHIP_DIR_SHOW == false ]] && return
 
-  _prompt_section \
-    "$SPACESHIP_DIR_COLOR" \
-    "$SPACESHIP_DIR_PREFIX" \
-    "%${SPACESHIP_DIR_TRUNC}~" \
-    "$SPACESHIP_DIR_SUFFIX"
+  _is_git
+  if [ $? -eq 0 ] && [ $SPACESHIP_DIR_TRUNC_IN_GIT == true ]; then
```

Comment:
I'm sorry. My bad, I didn't test it in non-git repo and now can confirm doesn't work. Will try new condition soon.